crates/rooch-framework/sources/session_key.move

```diff
@@ -107,20 +103,20 @@ module rooch_framework::session_key {
         if (!account_storage::global_exists<SessionKeys>(ctx, sender_addr)){
             return option::none()
         };
-        let auth_key = if(scheme == ed25519_validator::scheme()){
-            ed25519_validator::get_authentication_key_from_authenticator_payload(&authenticator_payload)
-        }else{
-            //TODO support other built-in validators
+        // We only support ed25519 validator for SessionKey now
+        if(scheme != ed25519_validator::scheme()){
             return option::none()
         };
+
+        let auth_key = ed25519_validator::get_authentication_key_from_authenticator_payload(&authenticator_payload);
```

Comment:
Is the method name too long?

Reply:
Yes. We can use self description.

Reply:
Or import the function directly and use it, eliminating the need to use `<handle>::`.